content/10-leetcode/00-code/00-py/two_pointer/lc_541_reverse_string.py

```
@@ -0,0 +1,18 @@
+@pytest.mark.parametrize(
+    ("input", "expected"),
+    (
+        (),
+    )
+)
+def test_some_function(input: any, expected: any) -> None:
+    pass
+
+
+if __name__ == "__main__":
+    test_some_function()
```

Pylint (3.3.7)

[error] No value for argument 'input' in function call

(E1120)